### PR TITLE
Feature roletoenv

### PR DESCRIPTION
**Pull request by essgje**
# \<Description\>

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by tommigitgaur — DISMISSED** [review]
LFTM

**Review by tommigitgaur — DISMISSED** [review]
LFTM

**Review by tommigitgaur — APPROVED** [review]
LFTM

File: apps/skilavottord/ws/src/environments/environment.ts [hunk 1]
```diff
@@ -24,5 +24,8 @@ export default {
     restUsername: process.env.FJARSYSLA_REST_USER,
     restPassword: process.env.FJARSYSLA_REST_PASS,
   },
+  skilavottord: {
+    userList: process.env.USER,
+  },
```
Comment (eirikurn) [inline apps/skilavottord/ws/src/environments/environment.ts:29]: This is breaking schemas.ts on main. Please pick a better env name, like "SKILAVOTTORD_USER_LIST". On many machines, the USER env is the name of the OS user, in my case "eirikur", which crashes when passed to JSON.parse().

You should also:

* Add this to the environment.prod.ts.
* Add a default value so we can generate schemas without crashing the process, eg `process.env.SKILAVOTTORD_USER_LIST ?? "[]"`.
* Document the format of this environment variable in README.md to help other devs get started with this project.

Comment (tommigitgaur) [reply apps/skilavottord/ws/src/environments/environment.ts:29]: We will fix that

Comment (tommigitgaur) [reply apps/skilavottord/ws/src/environments/environment.ts:29]: I have fixed this and changes are in main now.n

**Pull request merged**